integration

```diff
@@ -563,6 +563,7 @@
   "metbril/home-assistant-brandstofprijzen",
   "michaellunzer/Home-Assistant-Custom-Component-Fortnite",
   "midstar/heatmiser_wifi_ha",
+  "miguelangellv/tarifa_20td",
```

Comment:
<https://github.com/MiguelAngelLV/tarifa_20td/blob/main/hacs.json> country should be defined here.

Reply:
Added it